submissions/UA-girl/task_html-css-popup/css/styles.css

```diff
@@ -0,0 +1,252 @@
+/*Common*/
+
+* {
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    text-decoration: none;
+}
+
+body {
+    font-family: Arial, sans-serif;
+    font-size: 16px;
+    color: #333333;
+    box-sizing: border-box;
+}
+
+img {
+    max-width: 100%;
+    height: auto;
+}
+
+label, a {
+    color: #333333;
+    font-weight: bold;
+    box-sizing: border-box;
+    opacity: 0.5;
+    transition: 0.3s ease;
+}
+
+.container {
+    box-sizing: border-box;
+    max-width: 70%;
+    margin: 0 auto;
+}
+
+.hidden {
```

Comment:
Watch the [video ](https://www.youtube.com/watch?v=E6kLaaQFctU&t=413s) about hiding checkbox in the right way.